### PR TITLE
Fixed scroll input lag

### PR DESCRIPTION
**Pull request by Andrius-B**

### CODE REVIEW

**Comment by Andrius-B** [comment]
Illustrations of the problem before the fix:
lag_show
_The view is supposed to allways move with the touch pointer, but there is a noticable lag in the input_

And with the fix it works as expected _(and feels way better)_
no_lag_show

File: library/src/main/java/com/alamkanak/weekview/WeekView.java [hunk 1]
```diff
@@ -190,10 +190,17 @@
 
         @Override
         public boolean onDown(MotionEvent e) {
-            goToNearestOrigin();
+            stopScrolling();
+            //goToNearestOrigin();
```
Comment (jhoobergs) [inline library/src/main/java/com/alamkanak/weekview/WeekView.java]: Remove this comment

**Review by jhoobergs — CHANGES_REQUESTED** [review]

File: library/src/main/java/com/alamkanak/weekview/WeekView.java (hunk 1 above)
Comment (Andrius-B) [reply library/src/main/java/com/alamkanak/weekview/WeekView.java]: Yep, removed it

**Comment by reilem** [comment]
This PR is actually amazing. Sorry it took us this long to realise it. I will be merging and releasing today.

**Pull request merged**